docs/api_v0.yml

```diff
@@ -254,6 +255,10 @@ components:
           format: date-time
         user:
           $ref: "#/components/schemas/SharedUser"
+        reading_time:
```

Comment:
I manually edited these items - if this yaml file is machine generated rather than designed for human modification - I probably _didn't_ make upstream changes to the sources and need help.

Reply:
This file is human generated, no upstream changes required.